assets/server/realmadmin/_form_codes.html

```diff
@@ -87,6 +87,16 @@
     </div>
 
     {{if .features.EnableUserReport}}
+    <label>User report</label>
+
+    {{if and $realm.AllowsUserReport (not $realm.UseAuthenticatedSMS)}}
+    <div class="alert alert-warning" role="alert">
+      This realm has user report enabled, but authenticated SMS disabled. It is recommended
```

Comment:
"It is recommended to enable authenticated SMS is enabled as"

Reply:
fixed